chapters/ch41.md

```diff
@@ -40,11 +40,11 @@ Bây giờ, giả sử thuật toán của bạn đạt được:
 
 > This tells you that you have high avoidable bias on the training set. I.e., the algorithm is doing poorly on the training set. Bias reduction techniques should help.
 
-Điều này cho bạn biết rằng bạn có độ chệch có thể tránh được cao trên tập huấn luyện. Tức là, thuật toán đang hoạt động kém trên tập huấn luyện. Kỹ thuật giảm độ chệch sẽ có ích trong trường hợp này.
+Điều này cho bạn biết rằng bạn có độ chệch cao (có thể tránh được) trên tập huấn luyện. Tức là, thuật toán đang hoạt động kém trên tập huấn luyện. Kỹ thuật giảm độ chệch sẽ có ích trong trường hợp này.
```

Comment:
Mình nghĩ để như cũ ok rồi. Chẳng lẽ từ đó tiếng Việt khó hiểu đến mức mỗi khi nhắc đến là phải có mở ngoặc nữa sao.

Có thể thay đổi một chút nhưng tránh phải mở ngoặc mỗi khi nhắc tới từ đó.

Reply:
Mình nghĩ từ này có vấn đề ở chỗ. Hai nghĩa hoàn toàn khác nhau 
- Độ chệch tránh được cao --> "độ chệch tránh được" là một cụm danh từ. "Độ chệch tránh được" khác với "độ chệch" theo nghĩa chung. 
- Độ chệch cao tránh được --> độ chệch cao là một cụm danh từ. Ở đây ám chỉ độ chệch có giá trị cao. Và chỉ có đội có giá trị cao mới thường có thể tránh được. 

Ở đây là ý chỉ là ý thứ 2 mới đúng nên không viết như em @rootonchair  được.

Reply:
@ngcthuong  chương 22 là chương đầu tiên tác giả nói đến và định nghĩa "avoidable bias". Trong chương này thì "avoidable bias" có đứng riêng. Vậy nên em hiểu theo nghĩa 1.

Reply:
Thanks @rootonchair, mình đã check lại và sửa thành độ chệch tránh được cao